### PR TITLE
add homebrew formula for v0.2.0

### PR DESCRIPTION
**Pull request by saljam**
Adds a Homebrew formula to build v0.2.0 from source.
See #24.

### CODE REVIEW

**Comment by zchee** [comment]
@saljam Thanks PR.

What do you think is it to PR this Formula to Homebrew/homobrew?

File: Formula/docker-machine-driver-xhyve.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class DockerMachineDriverXhyve < Formula
+  desc "Xhyve driver for Docker Machine"
+  homepage "https://github.com/zchee/docker-machine-driver-xhyve"
+  url "https://github.com/zchee/docker-machine-driver-xhyve/archive/v0.2.0.tar.gz"
+  sha256 "affba9fe235e0ceb2711a52ea15b380bdcbec8aa2c106b54f750ede3c762dcb8"
+
+  depends_on "go" => :build
```
Comment (zchee) [inline Formula/docker-machine-driver-xhyve.rb]: I thought it was not enough

```ruby
depends_on "docker-machine"
```

File: Formula/docker-machine-driver-xhyve.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class DockerMachineDriverXhyve < Formula
+  desc "Xhyve driver for Docker Machine"
```
Comment (zchee) [inline Formula/docker-machine-driver-xhyve.rb]: I think `xhyve` is not uppercase

File: Formula/docker-machine-driver-xhyve.rb (hunk 1 above)
Comment (saljam) [inline Formula/docker-machine-driver-xhyve.rb]: Correct, thanks! I should probably add xhyve too.

Comment (zchee) [inline Formula/docker-machine-driver-xhyve.rb]: @saljam Thanks reply :)

No, `docker-machine-driver-xhyve` was embeded `xhyve` source.
Not need `xhyve` binary.

See https://github.com/zchee/docker-machine-driver-xhyve/releases/tag/v0.2.0 in Feature.

**Comment by zchee** [comment]
@saljam If you agree, I want to send PR to Homebrew/homebrew this PR.

What do you think?

File: Formula/docker-machine-driver-xhyve.rb (hunk 1 above)
Comment (saljam) [inline Formula/docker-machine-driver-xhyve.rb]: Ah, I missed that! Sounds good.

**Comment by saljam** [comment]
Yeah, I think it's OK to send this to Homebrew. It'll certainly bring more users + feedback.

**Comment by zchee** [comment]
@saljam Yes, I want more users and feedback.
I want to further improve based on its feedback.

Do you want to send a PR to homebrew? or me?

File: Formula/docker-machine-driver-xhyve.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    cd gopath/"src/github.com/zchee/docker-machine-driver-xhyve" do
+      system "go", "build", "-o", "docker-machine-driver-xhyve", "./bin/main.go"
+      bin.install "docker-machine-driver-xhyve"
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    docker-machine-driver-xhyve requires root privileges to correctly set up
+    networking. You can either use docker-machine with sudo, or change the
```
Comment (zchee) [inline Formula/docker-machine-driver-xhyve.rb]: `docker-machine` with `sudo` is buggy :(
e.g. If not exists `~/.docker/machine`, All of the `~/.docker/machine` folder under the file becomes root owner.

I think better to change the owner.

**Comment by saljam** [comment]
I'm happy for you to send it to Homebrew. Thanks!

**Comment by zchee** [comment]
@saljam OK!
I will send PR to homebrew.

Thanks for PR :)

**Comment by zchee** [comment]
@saljam I send PR has been merged!! :tada:
https://github.com/Homebrew/homebrew/pull/48002

and, Thanks for base code of the my pull-request :)

**Pull request closed without merging**